src/App.js

```diff
@@ -1,8 +1,9 @@
 import React, { useState } from "react";
 import "./App.css";
-import Header from "./components/UI/Header";
-import CardList from "./components/UI/CardList";
-import ViewOnly from "./components/UI/ViewOnly";
+import Header from "./components/UI/Header/Header";
+import CardList from "./components/CardList/CardList";
+import ViewOnly from "./components/ViewOnly/ViewOnly";
+import DeleteCards from "./components/DeleteCards/DeleteCards";
```

Comment:
Please use guide from the 'Understanding the basics' topic to follow the principles of components naming convention:
https://github.com/airbnb/javascript/tree/master/react
![image](https://user-images.githubusercontent.com/52723192/149239797-b59a0c83-3fbf-4809-948f-e30fd5229329.png)